components/private/Shapes/tests/unit/ShapesNoopTest.m

```diff
@@ -27,4 +27,23 @@ - (void)testInit {
   XCTAssertNotNil(view);
 }
 
+- (void)testCornerTreatmentEquality {
+  // Given
+  MDCCornerTreatment *cornerTreatment1 = [[MDCCornerTreatment alloc] init];
+  cornerTreatment1.valueType = MDCCornerTreatmentValueTypeAbsolute;
+  MDCCornerTreatment *cornerTreatment2 = [[MDCCornerTreatment alloc] init];
+  cornerTreatment2.valueType = MDCCornerTreatmentValueTypePercentage;
+
+  // Then
+  XCTAssertNotEqual(cornerTreatment1.hash, cornerTreatment2.hash);
```

Comment:
Please remove this `NotEqual` assert as well.

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/5954